HaloDAO/halodao-interface
# Features/hdev 269 migrate amm

## Description of Changes
Auto migration button from v1.0 to v1.1

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDEV-269)

## How To Test:
1 - if you're using the dev deployer you can test with the EURS/USDC pool. Make sure that the  migrate button of the pools with 0 rewards on v1 is disabled. If EURS/USDC LP staked in the v1.0 contract is 0, the migrate button should also be disabled.
2 - Stake some LP tokens by repeating this proposal https://defender.openzeppelin.com/#/admin/contracts/kovan-0xeD252AE0B697FbF60572eeFa1c7411E813A2f03B/proposals/005e1bcc-8843-44ec-aa74-a8f1da53bdb3
3 - Refresh/ Wait for the LP balance staked to be updated so activate the migrate button. 
4 - Click migrate and approve each txn (3 txns)


Note:
-  the console log at the bottom is for the API integration 
- this PR is pulled from @schystz where the pools are also already divided by versions


### Developer Checklist:


### Reviewers Checklist:
